Add a pallet tutorial bug

Due to some recent changes, the alpha.7 tutorial for adding pallets has two build errors:

```
error[E0425]: cannot find value `MILLICENTS` in this scope
   --> node/src/chain_spec.rs:129:28
    |
129 |             gas_price: 1 * MILLICENTS,
    |                            ^^^^^^^^^^ not found in this scope
    |
help: possible candidate is found in another module, you can import it into scope
    |
1   | use node_template_runtime::MILLICENTS;
    |

error[E0560]: struct `pallet_contracts::GenesisConfig` has no field named `gas_price`
   --> node/src/chain_spec.rs:129:13
    |
129 |             gas_price: 1 * MILLICENTS,
    |             ^^^^^^^^^ `pallet_contracts::GenesisConfig` does not have this field
    |
    = note: available fields are: `current_schedule`
```

The former is easily resolved using the suggested import, the latter will need other tweaks due to Gas no longer being a thing.

--- a/tuts/add-a-pallet/v2.0.0-alpha.7/index.md
+++ b/tuts/add-a-pallet/v2.0.0-alpha.7/index.md
@@ -522,7 +522,6 @@ fn testnet_genesis(initial_authorities: Vec<(AuraId, GrandpaId)>,
                     enable_println,
                     ..Default::default()
             },
-            gas_price: 1 * MILLICENTS,
         }),
         /*** End Added Block ***/
     }

--- a/tuts/start-a-private-network/v2.0.0-alpha.7/alicebob.md
+++ b/tuts/start-a-private-network/v2.0.0-alpha.7/alicebob.md
@@ -21,8 +21,9 @@ Alice (or whomever is playing her) should run these commands from node-template 
 ```bash
 # Purge any chain data from previous runs
 # You will be prompted to type `y`
-./target/release/node-template purge-chain
-
+./target/release/node-template purge-chain --base-path /tmp/alice --chain local
+```
+```bash
 # Start Alice's node
 ./target/release/node-template \
   --base-path /tmp/alice \
@@ -130,7 +131,9 @@ You should now see something like this.
 
 Now that Alice's node is up and running, Bob can join the network by bootstrapping from her node.
 His command will look very similar.
-
+```bash
+./target/release/node-template purge-chain --base-path /tmp/bob --chain local
+```
 ```bash
 ./target/release/node-template \
   --base-path /tmp/bob \

